rule.proto

```diff
@@ -54,6 +56,60 @@ message ChangeLineItemStatusParams {
   optional Status status = 3;
 }
 
+// The action parameters to change line items' bidding strategy.
+message ChangeLineItemBiddingStrategyParams {
+  enum BiddingStrategy {
+    UNSPECIFIED_STRATEGY = 0;
+    FIXED = 1;
+    MAXIMIZE_SPEND = 2;
+    PERFORMANCE_GOAL = 3;
+  }
+
+  // required
+  // The bidding strategy to change to.
+  optional BiddingStrategy bidding_strategy = 1;
+
+  // required
+  // The parameters used for each type of bidding strategy.
+  oneof params {
+    FixedBidStrategyParams fixed_bid_strategy_params = 2;
+    MaximizeSpendBidStrategyParams maximize_spend_bid_strategy_params = 3;
+    PerformanceGoalBidStrategyParams performance_goal_bid_strategy_params = 4;
+  }
+}
+
+// A message representing the parameters for fixed bid strategy.
+message FixedBidStrategyParams {
+  // required
+  // The fixed bid amount (micros) in the advertiser's currency.
+  optional int64 bid_amount_micros = 1;
+}
+
+enum BiddingStrategyPerformanceGoalType {
```

Comment:
I am curious how you find these BiddingStrategyPerformanceGoal?

Reply:
I looked at the DV360 Bid Strategy UI and referenced the [Public API documentation](https://developers.google.com/display-video/api/reference/rest/v1/BiddingStrategy#BiddingStrategyPerformanceGoalType). In the end, I only included a subset of the goal types.